package.json

```diff
@@ -3,14 +3,16 @@
   "version": "1.0.0-22",
   "description": "clojure.spec for Javascript",
   "main": "dist/js.spec.js",
+  "types": "types/index.d.ts",
   "scripts": {
     "build": "webpack --config webpack.config.js",
     "lint": "yarn run lint-src && yarn run lint-test",
     "lint-src": "eslint --config .eslintrc.json 'index.js' 'lib/**/*.js'",
     "lint-test": "eslint --config .eslintrc.test.json 'test/**/*.js'",
-    "prepare": "yarn run lint && yarn run build",
+    "prepare": "yarn run lint && yarn run test-ts && yarn run build",
```

Comment:
this hook should probably only just build the artifact (and it is executed on `npm install`) so we should leave it as it is

Reply:
I actually wonder why `yarn run build` is there.